host-spec/c06-consensus.tm

```diff
@@ -420,14 +528,15 @@
 
     \;
 
-    <math|H<rsub|BABE><around*|(|B|)>> must be included as a diegst item of
+    <math|H<rsub|BABE><around*|(|B|)>> must be included as a digest item of
     Pre-Runtime type in the header digest <math|H<rsub|d><around*|(|B|)>> as
     defined in Definition <reference|defn-digest>.\ 
   </definition>
 
   <\definition>
     <label|defn-block-signature><label|defn-babe-seal>The <strong|Block
-    Signature> noted by <math|S<rsub|B>> is computed as
+    Signature> <math|S<rsub|B>> is a signature of the block header hash (see
+    def.<reference|defn-block-header-hash>) and defined as
```

Comment:
We have been using (see Definition ...) instead of (see def.) and perhaps we should stick to that convention for the sake of consistency.